tests/unit/components/grapher/Grapher.spec.ts

```diff
@@ -67,6 +67,22 @@ describe('components/grapher/Grapher.vue', () => {
       expect(wrapper.contains('[data-test="grapher--grid-vertical"]')).toBeFalsy();
       expect(wrapper.contains('[data-test="grapher--grid-horizontal"]')).toBeFalsy();
     });
+
+    it('does not render yardlines if yardlines is false', () => {
+      store.commit('setYardlines', false);
+      expect(wrapper.contains('[data-test="grapher--yard-line"]')).toBeFalsy();
+    });
+
+    it('renders extra gridlines if yardlines is false', () => {
+      store.commit('setFourStepGrid', true);
+      store.commit('setYardlines', false);
+      expect(wrapper.findAll('[data-test="grapher--grid-vertical"]')).toHaveLength(7);
```

Comment:
Before this, can you check that the wrapper contains vertical lines? So it'd be

- Check for the existence of vertical lines
- Check that there are 7 vertical lines

Also, we can probably combine this test with the one above.

Reply:
^bump